### PR TITLE
attempt to add multiregioninig

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

**Comment by lc512k** [comment]
@wheresrhys we can do away with `/home/ubuntu/next-classification-api/server/policies/flag-enabled.js` as we've removed the flag already.

**Comment by wheresrhys** [comment]
ta. I've removed a bit of other apparently dead code too. Let me know if I overdid it

File: server/init.js [hunk 1]
```diff
@@ -28,9 +27,7 @@ app.get('/__access_metadata', (req, res) => {
 app.use('^/content/:id$', require('./controllers/access'));
 
 const port = process.env.PORT || 3000;
-const listen = app.listen(port, () => {
-	logger.info(`Listening on ${port}`);
-});
+const listen = app.listen(port);
```
Comment (wheresrhys) [inline server/init.js]: logging startup is done by n-express now

File: Makefile [hunk 2]
```diff
@@ -1,27 +1,17 @@
...
+	nht ship -m
 
 provision:
-	nbt provision ${TEST_APP}
-	nbt configure ft-next-classification-api ${TEST_APP} --overrides "NODE_ENV=branch"
-	nbt deploy ${TEST_APP} --skip-enable-preboot
+	nht float --testapp ${TEST_APP}
 
 tidy:
-	nbt destroy ${TEST_APP}
```
Comment (lc512k) [inline Makefile]: no `destroy`? Is this handled somewhere else?

Comment (wheresrhys) [inline Makefile]: yep - `nht float` does all the provision + tidy steps. I've suggested we should run a workshop on our dev tools as there are so many features of them that aren't widely known

**Comment by lc512k** [comment]
Only curious about where we `destroy` each test app now, other than that 👍

**Pull request merged**